src/index.js

```diff
@@ -421,6 +426,16 @@ L.Control.EasyPrint = L.Control.extend({
     if (show) return controlContainer.style.display = 'block';
     controlContainer.style.display = 'none';
   },
+  _toggleClasses: function (classesStr, show = false) {
```

Comment:
`show=false` is a default parameter which is an [es6 feature](https://googlechrome.github.io/samples/default-parameters-es6/), I presume the babel rollup would make this usable in the browser. 

Having said that you should be able to leave this blank and just do .
````
_toggleClasses: function (classesStr, show) {
````